client_test/helpers_test.go

```diff
@@ -229,3 +231,22 @@ func closeSimulatedChain(t *testing.T, chain chainservice.SimulatedChain) {
 		t.Fatal(err)
 	}
 }
+
+// generateMessageKey generates a ECDSA private key deterministically using the given pk bytes.
+func generateMessageKey(pk []byte) p2pcrypto.PrivKey {
+
+	// We use he given
+	totalSize := 256
+	large := make([]byte, totalSize)
```

Comment:
Curious, why is it necessary to create a 256 byte array to create a byte reader? Does `GenerateECDSAKeyPair` only accept 256 byte array readers?

Reply:
Makes sense, thanks for the explanation.

Reply:
Updated with a more thorough comment in https://github.com/statechannels/go-nitro/pull/1194/commits/2d0e31b528c6a3a37c78c09e68ddb0503ea61f69